doc/how_to/load_your_data_into_sorting.rst

```diff
@@ -0,0 +1,152 @@
+Load Your Own Data into a Sorting
+=================================
+
+Why make a :code:`Sorting`?
```

Comment:
As usual, your why sections are great!

Reply:
@JoeZiminski edited it so hats off for his assist.